util/progress/printer.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/moby/buildkit/util/progress/progressui"
 )
 
+const (
+	PrinterModeAuto  = "auto"
+	PrinterModeTty   = "tty"
+	PrinterModeQuiet = "quiet"
```

Comment:
defining those constant, should also create one for `plain` mode

Reply:
Yep! Done